bpf/lib/policy.h

```diff
@@ -186,7 +186,28 @@ __policy_can_access(const void *map, struct __ctx_buff *ctx, __u32 local_id,
 				return DROP_POLICY_DENY;
 			return policy->proxy_port;
 		}
+		
+		/* L4-only lookup, If not fill out port, podSelector and namespaceSelector */
+		key.dport = 0;
+		policy = map_lookup_elem(map, &key);
+		if (likely(policy)) {
+			account(ctx, policy);
+			*match_type = POLICY_MATCH_L4_ONLY;
+			if (unlikely(policy->deny))
+				return DROP_POLICY_DENY;
+			return policy->proxy_port;
+		}
+		
+		/* L4-only lookup, If not fill out port ,but use podSelector and namespaceSelector */
```

Comment:
Since the remote_id is in the key for this lookup, it is not an L4-only lookup, but an L3/L4 one: even though the port is wildcarded, the protocol is not. This variant of an L3/L4 lookup should be done before the L4-only lookups above, and presumably after the "full" L3/L4 lookup.

Reply:
done